### PR TITLE
Feature/init

### PR DESCRIPTION
**Pull request by rchengvariant**
# Initial Commit:

.gitignore (Get from gitignore.io - Create Useful .gitignore Files For Your Project (toptal.com))
.gitattributes (Get from gitattributes.io - Create .gitattributes file for your project)
README.md
.github

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+# actions-collection
+
+## Description
+
+Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet). 
+
+To run shell functions from collection.sh:
+
+```bash
+sh collection.sh {function}
```
Comment (vibindaniel) [inline README.md:10]: Shouldn't it be

```bash
source collection.sh
{function}
```

**Review by vibindaniel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rchengvariant) [reply README.md:10]: sorry i'll change it. It should be:
```
source collection.sh
sh collection.sh docker_publish
```

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
...
+
+## Description
+
+Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet)
+
+To run shell functions from collection.sh:
+
+```bash
+source collection.sh
+sh collection.sh docker_publish
```
Comment (vibindaniel) [inline README.md:11]: Can't you simply do `docker_publish` once you do `source`?

**Review by vibindaniel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rchengvariant) [reply README.md:11]: I think so but I feel like if we don't specify the file: the memory might (shouldn't) have another function named docker_publish.

Comment (rchengvariant) [reply README.md:11]: but im okay with just ```docker_publish```

Comment (vibindaniel) [reply README.md:11]: The memory shouldn't be having that. If it has, then we are doing something wrong somewhere.

Comment (rchengvariant) [reply README.md]: Fixed.

**Review by vibindaniel — APPROVED** [review]

**Pull request merged**